exercises/practice/zebra-puzzle/zebra-puzzle.js

```diff
@@ -0,0 +1,11 @@
+//
+// This is only a SKELETON file for the 'Zebra Puzzle' exercise. It's been provided as a
+// convenience to get you started writing code faster.
+//
+
+export function zebraPuzzle() {
+  return {
+    waterDriker: 'Englishman | Spaniard | Ukrainian | Norwegian | Japanese',
+    zebraOwner: 'Englishman | Spaniard | Ukrainian | Norwegian | Japanese',
```

Comment:
I think it would be better to explain the response format in an instructions.append.md file, see this one from the Go track as an example: https://github.com/exercism/go/blob/main/exercises/practice/zebra-puzzle/.docs/instructions.append.md